doc/docs/Python_Tutorials/Near_to_Far_Field_Spectra.md

```diff
@@ -381,15 +381,15 @@ if __name__ == '__main__':
 
-The simulation consists of an $E_r$ point-dipole source ($\lambda$ = 1.0 μm) at $r$ = 0.6 μm embedded within a disc (radius of 1.2 μm) of index $n$ = 2.4 above a perfect-metallic ground plane. This is similar to the configuration in [Tutorial/Extraction Efficiency of a Light-Emitting Diode (LED)](Local_Density_of_States.md#extraction-efficiency-of-a-light-emitting-diode-led). Unlike the infinitely extended slab of the LED, a *finite* structure such as the disc ensures that all the power from the dipole emitter is radiated. The LED contains waveguide modes which are more challenging to disentagle from the radiated power.
+The simulation consists of an $E_r$ "ring"-current source with $\lambda$ = 1.0 μm and radius of 0.6 μm embedded within a disc of radius 1.2 μm, refractive index $n$ = 2.4, and thickness $0.7 \lambda / n$. The disc is positioned above a perfect-metallic ground plane. The source has azimuthal dependence $e^{im\phi}$ with $m = -1$, which leads to circularly polarized fields (unlike $|m| \ne 1$). Unlike the [infinitely extended slab of the LED](Local_Density_of_States.md#extraction-efficiency-of-a-light-emitting-diode-led), a *finite* structure such as the disc ensures that all the power from the source is radiated away. The LED contains waveguide modes which are more challenging to disentagle from the radiated power.
 
 A schematic of the simulation layout is shown below. The flux and near-field monitors (shown in blue) are overlapping.
 
 ![](../images/disc_radiation_layout.png#center)
 
-Obtaining the radiation pattern $P(\theta)$ of the disc involves computing the radial (or "outgoing") flux from the far fields along the circumference of a quarter circle (i.e. angular range of $[0, \pi/2]$). The radius $r$ of the circle needs to be sufficiently large ($\gg \lambda$) to ensure accurate results but is otherwise arbitrary. The total flux is then computed by integrating $P(\theta)$ over the surface of a hemisphere with radius $r$ in [spherical coordinates](https://en.wikipedia.org/wiki/Spherical_coordinate_system):
+Obtaining the radiation pattern $P(\theta)$ of the disc involves computing the radial (or "outgoing") flux from the far fields along the circumference of a quarter circle with angular range of $[0, \pi/2]$. Note that the radiation pattern $P(\theta)$ of circularly polarized fields is independent of $\phi$. The radius $r$ of the far-field circle needs to be sufficiently large ($\gg \lambda$) to ensure accurate results but is otherwise arbitrary. The total flux is then computed by integrating $P(\theta)$ over the surface of a hemisphere with radius $r$ in [spherical coordinates](https://en.wikipedia.org/wiki/Spherical_coordinate_system):
```

Comment:
```suggestion
Obtaining the radiation pattern $P(\theta)$ of the disc involves computing the radial (or "outgoing") flux from the far fields along the circumference of a quarter circle with angular range of $[0, \pi/2]$. Note that the radiation pattern $P(\theta)$ of any fields $\sim e^{im\phi}$ (for any $m$) is independent of $\phi$. The radius $r$ of the far-field circle needs to be sufficiently large ($\gg \lambda$) to ensure accurate results but is otherwise arbitrary. The total flux is then computed by integrating $P(\theta)$ over the surface of a hemisphere with radius $r$ in [spherical coordinates](https://en.wikipedia.org/wiki/Spherical_coordinate_system):
```